src/data/load_geometry.js

```diff
@@ -39,6 +39,8 @@ export default function loadGeometry(feature: VectorTileFeature): Array<Array<Po
 
             if (point.x < bounds.min || point.x > bounds.max || point.y < bounds.min || point.y > bounds.max) {
                 warnOnce('Geometry exceeds allowed extent, reduce your vector tile buffer size');
+                point.x = clamp(point.x, bounds.min, bounds.max);
```

Comment:
Will clamping here change the current behavior (especially with respect to geometries split across tile boundaries? Or is it always broken when points are beyond the bounds?

Reply:
It's broken. It's just that the bounds were increased 2x when we switched to bigger line layout, and got back when we reverted that. The clamping at minimum doesn't get things worse, but also often makes things better.